homeassistant/components/smarty/__init__.py

```diff
@@ -0,0 +1,65 @@
+"""Support to control a Salda Smarty XP/XV ventilation unit."""
+
+from datetime import timedelta
+import ipaddress
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (CONF_NAME, CONF_HOST)
+from homeassistant.helpers import discovery
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import dispatcher_send
+from homeassistant.helpers.event import track_time_interval
+
+REQUIREMENTS = ['pysmarty==0.8']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'smarty'
```

Comment:
Move this constant up under requirements so it's easier to findthe HASS specific ones in a single grouping.